### DISCUSSION TITLE
How hard would it be to support the gpodder API?

### DISCUSSION THREAD

**Discussion by LivInTheLookingGlass** [FreshRSS/FreshRSS#5337] [Ideas]
It seems like this would be a good target. And it would enable support for synchronizing to e.g. AntennaPod

This would probably need to be done as a system extension, and would probably need to write to /api, which is a problem. I do think it would be worth it, though

**Comment by Frenzie** [comment]
Reference: https://gpoddernet.readthedocs.io/en/latest/api/
